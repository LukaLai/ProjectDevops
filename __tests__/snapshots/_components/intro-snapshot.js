import { render } from '@testing-library/react'
import { Intro } from '../../../src/app/_components/intro'

it('renders homepage unchanged', () => {
    const { container } = render(<Intro />)
    expect(container).toMatchSnapshot()
})