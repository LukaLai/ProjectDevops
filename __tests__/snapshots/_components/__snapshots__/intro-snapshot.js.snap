// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <section
    class="flex-col md:flex-row flex items-center md:justify-between mt-16 mb-16 md:mb-12"
  >
    <h1
      class="text-5xl md:text-8xl font-bold tracking-tighter leading-tight md:pr-8"
    >
      Blog.
    </h1>
    <h4
      class="text-center md:text-left text-lg mt-5 md:pl-8"
    >
      A statically generated blog example using
       
      <a
        class="underline hover:text-blue-600 duration-200 transition-colors"
        href="https://nextjs.org/"
      >
        Next.js
      </a>
       
      and 
      Markdown
      .
    </h4>
  </section>
</div>
`;
